salt/kubernetes-master/deploy_addons.sh

```diff
@@ -25,15 +25,36 @@ else
 fi
 
 echo "Deploying DNS on Kubernetes"
-KUBEDNS=`eval "$KUBECTL_PATH $NS_ARGS $SRV_ARGS get rc | grep kube-dns | cat"`
+
```

Comment:
As per our conversation, I will open a bugreport to backport the required kubernetes feature to our salt packages and once done will reopen a PR to replace the bash script. For now, I would like to get this PR merged.